migration/iD10/oracle/202209121558_IDEMPIERE-5418.sql

```diff
@@ -0,0 +1,252 @@
+;
+
+-- Sep 12, 2022, 3:58:57 PM CEST
+UPDATE AD_Table SET IsChangeLog='N',Updated=TO_TIMESTAMP('2022-09-12 15:58:57','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Table_ID=200361
+;
+
+-- Sep 12, 2022, 3:59:59 PM CEST
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,Description,Help,AD_Table_ID,AD_Val_Rule_ID,ColumnName,DefaultValue,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,SeqNo,IsEncrypted,AD_Reference_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,EntityType,IsSyncDatabase,IsAlwaysUpdateable,IsAutocomplete,IsAllowLogging,AD_Column_UU,IsAllowCopy,SeqNoSelection,IsToolbarButton,IsSecure,FKConstraintType,IsHtml) VALUES (215305,0,'Tenant','Tenant for this installation.','A Tenant is a company or a legal entity. You cannot share data between Tenants.',200361,129,'AD_Client_ID','@#AD_Client_ID@',22,'N','N','Y','N','N',0,'N',19,0,0,'Y',TO_TIMESTAMP('2022-09-12 15:59:59','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2022-09-12 15:59:59','YYYY-MM-DD HH24:MI:SS'),100,102,'N','N','D','N','N','N','Y','9507cdcf-8a0b-44a1-9776-5fafd6b2cb56','N',0,'N','N','D','N')
+;
+
+-- Sep 12, 2022, 4:00:02 PM CEST
+CREATE TABLE AD_ChartDatasource_Trl (AD_Client_ID NUMBER(10) NOT NULL)
+;
+
+-- Sep 12, 2022, 4:00:37 PM CEST
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,AD_Table_ID,ColumnName,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,SeqNo,IsEncrypted,AD_Reference_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,EntityType,IsSyncDatabase,IsAlwaysUpdateable,IsAutocomplete,IsAllowLogging,AD_Column_UU,IsAllowCopy,SeqNoSelection,IsToolbarButton,IsSecure,IsHtml) VALUES (215306,0,'Chart Datasource',200361,'AD_ChartDatasource_ID',22,'N','Y','Y','N','N',0,'N',30,0,0,'Y',TO_TIMESTAMP('2022-09-12 16:00:36','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2022-09-12 16:00:36','YYYY-MM-DD HH24:MI:SS'),100,54269,'N','N','D','N','N','N','Y','2d9f86a9-ce57-45b2-932b-4610818f99e7','Y',0,'N','N','N')
+;
+
+-- Sep 12, 2022, 4:00:40 PM CEST
+UPDATE AD_Column SET IsUpdateable='N', FKConstraintName='ADChartDatasource_ADChartDatas', FKConstraintType='N',Updated=TO_TIMESTAMP('2022-09-12 16:00:40','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=215306
+;
+
+-- Sep 12, 2022, 4:00:40 PM CEST
+ALTER TABLE AD_ChartDatasource_Trl ADD AD_ChartDatasource_ID NUMBER(10) NOT NULL
+;
+
+-- Sep 12, 2022, 4:00:40 PM CEST
+ALTER TABLE AD_ChartDatasource_Trl ADD CONSTRAINT ADChartDatasource_ADChartDatas FOREIGN KEY (AD_ChartDatasource_ID) REFERENCES ad_chartdatasource(ad_chartdatasource_id) DEFERRABLE INITIALLY DEFERRED
```

Comment:
changing the FK to cascade must change this statement too - both in oracle and postgresql